atmosphere_unit_service/service/domain/device.py

```diff
@@ -0,0 +1,29 @@
+from service.domain.pressure_stabilizer.pressure_stabilizer import PressureStabilizer
+from service.domain.temperature_stabilizer.temperature_stabilizer import TemperatureStabilizer
+from service.domain.humidity_manager.humidity_manager import HumidityManager
+
+import uuid
+
+class Device:
+
+    def __init__(self, name: str = "device") -> None:
+        self.name = name
+
+        self.is_on = False
+        self.alarm = False
+        self.durability = 100.0
+
+        self.pressure_stabilizer = PressureStabilizer(name=f"PressureStabilizer-{uuid.uuid4()}")
+        self.temperature_stabilizer = TemperatureStabilizer(name=f"TemperatureStabilizer-{uuid.uuid4()}")
+        self.humidity_manager = HumidityManager(name-f"HumidityManager-{uuid.uuid4}")
```

Comment:
Сломанная конструкция

```suggestion
        self.humidity_manager = HumidityManager(name=f"HumidityManager-{uuid.uuid4}")
```